appveyor.yml

```diff
@@ -0,0 +1,56 @@
+
+version: 0.5.{build}
+
+branches:
+  only:
+    - master
+
+max_jobs: 1
+
+image:
+- Visual Studio 2015
+- Visual Studio 2017
```

Comment:
So you build with 2 versions of visual studio? 
[Here](https://cowboyprogrammer.org/2014/06/building-python-wheels-for-windows/) they say that 2010 is the most important to be compatible with. But is this feasible with C++11?

Reply:
that article is almost four years old, but there is a VS 2013 image, so we can do all three

Reply:
dist utils complains requiring MSVC > 2014 - so just the two it seems